hedera-node/docs/design/services/smart-contract-service/IHFC-associate-dissociate.md

```diff
@@ -0,0 +1,81 @@
+# IHRC Facade Associate and Dissociate Functionality
```

Comment:
Shouldn't this be a HIP?

Reply:
Yes there should be a HIP but things were done out of order this time since there could be a bit of a rush to get this functionality out because new contracts do not have the ability to associate tokens with the new security measures